Fhi.HelseId.Blazor/Fhi.HelseId.Blazor.csproj

```diff
@@ -37,7 +37,10 @@
     <PackageReference Include="Refit" Version="6.3.2" />
     <PackageReference Include="Refit.HttpClientFactory" Version="6.3.2" />
     <PackageReference Include="Microsoft.AspNetCore.HeaderPropagation" Version="6.0.24" />
-    <PackageReference Include="Fhi.HelseId" Version="5.0.4" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Fhi.HelseId\Fhi.HelseId.csproj" />
```

Comment:
Du trenger en pack ref her, men sett versjonsnr til 5.*,  da slipper vi å oppdatere for hver eneste lille endring i Fhi.HelseId.